src/styles/color-palette.scss

```diff
@@ -0,0 +1,8 @@
+$primary-color: #1d3557;
+$error-color: #e63946;
+$bg-color: #f1faee;
+$text-color: #293241;
+$color-white: #fff;
+$color-green: #8fffab;
+
+$border-radius: 4px;
```

Comment:
I would suggest specific name, since there may be different border radiuses.
Or if it is described in the project, you can use `border-radius-1`, `...-2`, etc... with values `2px`, `4px`, etc...
You can check [Tailwind](https://tailwindcss.com/docs/border-radius) for the example.